pkg/cmd/browse/browse.go

```diff
@@ -0,0 +1,250 @@
+package browse
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/api"
+	"github.com/cli/cli/internal/ghrepo"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type browser interface {
+	Browse(string) error
+}
+
+type BrowseOptions struct {
+	HttpClient func() (*http.Client, error)
+	IO         *iostreams.IOStreams
+	BaseRepo   func() (ghrepo.Interface, error)
+	Browser    browser
+
+	SelectorArg   string
+	AdditionalArg string
+
+	ProjectsFlag bool
+	WikiFlag     bool
+	SettingsFlag bool
+	BranchFlag   bool
+	//LineFlag     bool
+}
+
+type exitCode int
+
+const (
+	exitSuccess      exitCode = 0
+	exitNotInRepo    exitCode = 1
+	exitTooManyFlags exitCode = 2
+	exitTooManyArgs  exitCode = 3
+	exitExpectedArg  exitCode = 4
+	exitInvalidCombo exitCode = 5
+)
+
+func NewCmdBrowse(f *cmdutil.Factory) *cobra.Command {
+
+	opts := &BrowseOptions{
+		IO:         f.IOStreams,
+		HttpClient: f.HttpClient,
+		Browser:    f.Browser,
+		BaseRepo:   f.BaseRepo,
+	}
+
+	cmd := &cobra.Command{
+		Long:  "Work with GitHub in the browser",       // displays when you are on the help page of this command
```

Comment:
Not 100% sure about this wording, but just trying to get at a more specific description of what we're trying to get at here.
```suggestion
		Long:  "Open specific pages in the browser on GitHub",       // displays when you are on the help page of this command
```

Reply:
Ok I think we have a good revised edition of the help page including the `Long:` text. It's at the top with the revised pull request description!